integrations/QuoraPixel/browser.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable */
+import logger from "../../utils/logUtil";
+import { NAME } from "./constants";
+import { getHashFromArray } from "../utils/commonUtils";
+
+class QuoraPixel {
+  constructor(config) {
+    this.name = NAME;
+    this.pixelId = config.pixelId;
+    this.eventsToQPEvents = config.eventsToQPEvents;
+  }
+
+  loadScript() {
+    !(function (q, e, v, n, t, s) {
+      if (q.qp) return;
+      n = q.qp = function () {
+        n.qp ? n.qp.apply(n, arguments) : n.queue.push(arguments);
+      };
+      n.queue = [];
+      t = document.createElement(e);
+      t.async = !0;
+      t.src = v;
+      s = document.getElementsByTagName(e)[0];
+      s.parentNode.insertBefore(t, s);
+    })(window, "script", "https://a.quora.com/qevents.js");
+    window.qp("init", this.pixelId);
+    window.qp("track", "ViewContent");
```

Comment:
can we test that if viewContent fires any specific event or not? Or if we reframe it, if the CDN loads properly if we do not load `ViewContent` by default? If it can be loaded, then `ViewContent` can added as an option in the webapp as well.

Reply:
ViewContent is the part of `Base Pixel`. It is like a `Page View `event. We can load the Quora CDN without including it.
<img width="1714" alt="Screenshot 2022-09-12 at 2 46 12 PM" src="https://user-images.githubusercontent.com/60897972/189617539-1901ad1e-d00c-42f1-9a55-191563fcc899.png">